src/priya/A21_StringDigit.java

```diff
@@ -0,0 +1,20 @@
+package priya;
+
+public class A21_StringDigit {
+	
+	void printDigit() {
```

Comment:
if this variable is common for other methods then we should write it at the instance level.